mapper/census_filter_outputs.go

```diff
@@ -59,20 +59,33 @@ func CreateCensusFilterOutputsPage(ctx context.Context, req *http.Request, baseP
 
 	// DIMENSIONS
 	p.DatasetLandingPage.Dimensions = mapFilterOutputDims(fDims, queryStrValues, req.URL.Path, p.DatasetLandingPage.IsMultivariate)
-	coverage := []sharedModel.Dimension{
-		{
-			IsCoverage:        true,
-			IsDefaultCoverage: hasNoAreaOptions,
-			Title:             Coverage,
-			Name:              strings.ToLower(Coverage),
-			ID:                strings.ToLower(Coverage),
-			Values:            fDims[0].Options,
-			ShowChange:        true,
-		},
+	coverage := sharedModel.Dimension{
+		IsCoverage:        true,
+		IsDefaultCoverage: hasNoAreaOptions,
+		Title:             Coverage,
+		Name:              strings.ToLower(Coverage),
+		ID:                strings.ToLower(Coverage),
+		Values:            fDims[0].Options,
+		ShowChange:        true,
+	}
+	area := p.DatasetLandingPage.Dimensions[0]
+	displayedDims := p.DatasetLandingPage.Dimensions[1:]
+	sort.Slice(displayedDims, func(i, j int) bool {
+		return displayedDims[i].Title < displayedDims[j].Title
+	})
+	p.DatasetLandingPage.Dimensions = append([]sharedModel.Dimension{pop, area, coverage}, displayedDims...)
+
+	// CUSTOM TITLE
+	if helpers.IsBoolPtr(filterOutput.Custom) || p.DatasetLandingPage.IsMultivariate {
+		nonGeoDims := getNonGeographyDims(&p.DatasetLandingPage.Dimensions)
+		title := buildConjoinedList(nonGeoDims, true)
+		vDims := getNonGeographyVersionDims(&version.Dimensions)
+		vTitle := buildConjoinedList(vDims, true)
+		if vTitle != title {
+			p.Metadata.Title = strings.ToUpper(title[:1]) + strings.ToLower(title[1:])
+			p.DatasetLandingPage.IsCustom = true
+		}
```

Comment:
This needs to account for the case where the default dataset has non-geog dimensions X, Y and the user has come the build your own journey and added X, Y independently. 

Short version if IsCustom always use the custom title
```
if vTitle != title || helpers.IsBoolPtr(filterOutput.Custom) {
	p.Metadata.Title = strings.ToUpper(title[:1]) + strings.ToLower(title[1:])
	p.DatasetLandingPage.IsCustom = true
}
```

Reply:
Fair enough although I thought the default datasets were not going to have any non-geog dimensions?